app/controllers/members_controller.rb

```diff
@@ -24,6 +24,20 @@ def create
     end
   end
 
+  def confirm_email
+    @member = Member.find_by_confirm_token(params[:id])
+    if @member
+      @member.email_activate
```

Comment:
For actions like this, rename `email_activate` to `verify_email!` (with a bang), to be consistent.